main.py

```diff
@@ -137,7 +160,7 @@ async def command_stop():
         await bot_member.edit(nick=original_bot_nickname)
 
 
-async def command_play(message: Message):
+async def command_play(message: Message, skip_count=0):
```

Comment:
Please add types. They help IDEs give better intellisense, thus preventing bugs!

```suggestion
async def command_play(message: Message, skip_count: int = 0):
```